src/applications/static-pages/facilities/ExpandableVamcOperatingStatus.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
```

Comment:
filename and location seems misnamed.    THis is supplemental status not operating status and going one level up it is really the pattern for alert expandabile.